### ISSUE TITLE
Is this project alive? 

### ISSUE THREAD

**Issue by johntrandall** [lovato/gcontacts-sync#1]
Was any progress ever made on this project? I have a similar need.

**Comment by lovato** [comment]
Hi! My apologies. I just saw this today.
I did actually write some code, and was using it. I can share it with you.
I did not add it here due to hardcoded passwords in it.

My goal is to share android contacts with wife at that time, but today she uses an iPhone, so it was abandoned.

**Comment by lovato** [comment]
Yep, I found it. Not used since 2014. But I have access to all the code. Python 2.7 was used.
